app/src/main/java/com/hedvig/app/ApplicationModule.kt

```diff
@@ -606,6 +610,8 @@ val useCaseModule = module {
     single { GraphQLQueryUseCase(get()) }
     single { GetCrossSellsUseCase(get(), get()) }
     single { CreateOtpAttemptUseCase(get()) }
+    single { SendOtpCodeUseCaseImpl(get()) }
+    single { ReSendOtpCodeUseCaseImpl(get()) }
```

Comment:
```suggestion
    single<SendOtpCodeUseCase> { SendOtpCodeUseCaseImpl(get()) }
    single<ReSendOtpCodeUseCase> { ReSendOtpCodeUseCaseImpl(get()) }
```

Isn't this necessary in order to bind against the `interface`? 🤔